enterprise_access/apps/subsidy_access_policy/tests/test_models.py

```diff
@@ -76,8 +76,8 @@ def setUp(self):
         )
         self.mock_catalog_contains_content_key = catalog_contains_content_key_patcher.start()
 
-        get_content_metadata_patcher = patch.object(
-            SubsidyAccessPolicy, 'get_content_metadata'
+        get_content_metadata_patcher = patch(
+            'enterprise_access.apps.subsidy_access_policy.models.get_and_cache_content_metadata'
```

Comment:
Better to patch the thing underlying the policy model's `get_content_metadata` method.